src/libs/ReportUtils.ts

```diff
@@ -3916,6 +3917,88 @@ const reportNameCache = new Map<string, {lastVisibleActionCreated: string; repor
  */
 const getCacheKey = (report: OnyxEntry<Report>): string => `${report?.reportID}-${report?.lastVisibleActionCreated}-${report?.reportName}`;
 
+/**
+ * Get the title for a report that has no parentReportAction (e.g. group chat, policy expense chat, 1:1 DM, etc.)
+ */
+function getRootLevelReportName({
+    report,
+    policy,
+    personalDetails,
+    invoiceReceiverPolicy,
+}: {
+    report: OnyxEntry<Report>;
+    policy?: OnyxEntry<Policy>;
+    personalDetails?: Partial<PersonalDetailsList>;
+    invoiceReceiverPolicy?: OnyxEntry<Policy>;
+}): string {
+    const reportID = report?.reportID;
+    const cacheKey = getCacheKey(report);
+
+    let formattedName: string | undefined;
+
+    if (isClosedExpenseReportWithNoExpenses(report)) {
+        return Localize.translateLocal('parentReportAction.deletedReport');
+    }
```

Comment:
A closed expense report with no expenses isn't a root level report right? Expense reports always have a parent chat report. So I think this should stay in getReportName.

Reply:
In general there are a lot of cases in here that are not root level reports, so I'm a bit confused about the goals of this function. Can we narrow down the scope much more? It's only possible to share to a few destination report types now, correct?